schemas.py

```diff
@@ -36,3 +36,9 @@ class AddressSchema(Schema):
     postal_code = fields.Str(required=True, validate=check_empty_str)
     local_address = fields.Str(required=True, validate=check_empty_str)
     phone = fields.Str(required=True, validate=check_empty_str)
+
+
+class PictureSchema(Schema):
+    uuid = fields.UUID(required=True, dump_only=True)
+    title = fields.Str(required=True, validate=check_empty_str, dump_only=True)
+    extension = fields.Str(required=True, validate=check_empty_str, dump_only=True)
```

Comment:
Nella ItemPictureResource stiamo prendendo in input title e extension, quindi direi che questi `dump_only` non ci vogliono.